packages/git-raw-commits/test.js

```diff
@@ -192,4 +190,20 @@ describe('git-raw-commits', function () {
         done()
       }))
   })
+
+  it('should emit an error if git is not available', function (done) {
+    var rawExecOpts = { env: process.env }
+    rawExecOpts.env.PATH = ''
+
+    gitRawCommits({}, rawExecOpts)
+      .on('error', function (err) {
+        expect(err).to.be.ok // eslint-disable-line no-unused-expressions
+        done()
+      })
+      .pipe(through(function () {
+        done('should error')
+      }, function () {
```

Comment:
Here the flush function is _not_ called before `error` is emitted.  In this case there is no output whatsoever (empty or otherwise) so output processing is apparently skipped.

This is inconsistent with the above comment that the flush function is called before `error` is emitted, but I can't think of another case where this might happen.